lldb/source/Plugins/Language/Swift/CMakeLists.txt

```diff
@@ -27,6 +33,7 @@ add_lldb_library(lldbPluginSwiftLanguage PLUGIN
     lldbPluginTypeSystemSwift
     swiftAST
     swiftClangImporter
+    ${SWIFT_CORE_LIB} # fixes a linker issue when building on Windows.
```

Comment:
It wouldn't hurt to be more verbose and mention which symbol is missing, and that we don't actually think this is the correct solution, but merely a workaround.

Reply:
Made the comment more explicit 👍

Reply:
This really is the right solution - if you have Swift code in the image, you need to have `swiftrt.obj` linked in to register the metadata with the runtime, and thus you need to register the runtime.

If this is truly a workaround, then we should probably indicate what dependency we are pulling in that we should not be which is resulting in Swift code being pulled into the module.

Reply:
> If this is truly a workaround, then we should probably indicate what dependency we are pulling in that we should not be which is resulting in Swift code being pulled into the module.

Could it be linked to these commits? https://github.com/swiftlang/swift/commit/8cb86acab5a99f1163bbef4ea37ac43d743b18d6 -  https://github.com/swiftlang/swift/commit/0c42b5796267f52a5b5bf312b457addf927f6c51?

I'm not sure how to determine if it's a workaround or the right solution.

Reply:
No, neither of those are related. The first is aa change to the stubs, and the second is limited to ELF. The way to determine if this is a workaround or the right solution is to understand _why_ there is any Swift code in the module. If you are supposed to have Swift code, then this is the right fix - Swift code generally requires that you are linking to swiftCore as that is what provides the implementation for the basic types.